enterprise/models.py

```diff
@@ -2235,13 +2240,28 @@ class Meta:
         'placeholder {course_name} will be replaced with the name of the course or program that was enrolled in.'
     )
 
+    template_type_choices = [
+        (DEFAULT_ENROLL_EMAIL_TEMPLATE_TYPE, 'Default Enrollment Template'),
```

Comment:
ah yeah good call. I forgot to rename these. The default name does not really make sense here.. we just happen to use it as default in the code flows here. renamed to 'SELF_' instead of 'default_', and strings accordingly too